exporters/zipkin/src/main/java/io/opentelemetry/exporter/zipkin/ZipkinSpanExporter.java

```diff
@@ -47,188 +31,28 @@ public final class ZipkinSpanExporter implements SpanExporter {
-        Enumeration<InetAddress> addresses = nic.getInetAddresses();
-        while (addresses.hasMoreElements()) {
-          InetAddress address = addresses.nextElement();
-          if (address.isSiteLocalAddress()) {
-            return address;
-          }
-        }
-      }
-    } catch (Exception e) {
-      // don't crash the caller if there was a problem reading nics.
-      baseLogger.log(Level.FINE, "error reading nics", e);
-    }
-    return null;
-  }
-
-  Span generateSpan(SpanData spanData) {
-    Endpoint endpoint = getEndpoint(spanData);
 
-    long startTimestamp = toEpochMicros(spanData.getStartEpochNanos());
-    long endTimestamp = toEpochMicros(spanData.getEndEpochNanos());
-
-    Span.Builder spanBuilder =
-        Span.newBuilder()
-            .traceId(spanData.getTraceId())
-            .id(spanData.getSpanId())
-            .kind(toSpanKind(spanData))
-            .name(spanData.getName())
-            .timestamp(toEpochMicros(spanData.getStartEpochNanos()))
-            .duration(Math.max(1, endTimestamp - startTimestamp))
-            .localEndpoint(endpoint);
-
-    if (spanData.getParentSpanContext().isValid()) {
-      spanBuilder.parentId(spanData.getParentSpanId());
-    }
-
-    Attributes spanAttributes = spanData.getAttributes();
-    spanAttributes.forEach(
-        (key, value) -> spanBuilder.putTag(key.getKey(), valueToString(key, value)));
-    int droppedAttributes = spanData.getTotalAttributeCount() - spanAttributes.size();
-    if (droppedAttributes > 0) {
-      spanBuilder.putTag(OTEL_DROPPED_ATTRIBUTES_COUNT, String.valueOf(droppedAttributes));
-    }
-
-    StatusData status = spanData.getStatus();
-
-    // include status code & error.
-    if (status.getStatusCode() != StatusCode.UNSET) {
-      spanBuilder.putTag(OTEL_STATUS_CODE, status.getStatusCode().toString());
-
-      // add the error tag, if it isn't already in the source span.
-      if (status.getStatusCode() == StatusCode.ERROR && spanAttributes.get(STATUS_ERROR) == null) {
-        spanBuilder.putTag(STATUS_ERROR.getKey(), nullToEmpty(status.getDescription()));
-      }
-    }
-
-    InstrumentationScopeInfo instrumentationScopeInfo = spanData.getInstrumentationScopeInfo();
-
-    if (!instrumentationScopeInfo.getName().isEmpty()) {
-      spanBuilder.putTag(KEY_INSTRUMENTATION_SCOPE_NAME, instrumentationScopeInfo.getName());
-      // Include instrumentation library name for backwards compatibility
-      spanBuilder.putTag(KEY_INSTRUMENTATION_LIBRARY_NAME, instrumentationScopeInfo.getName());
-    }
-    if (instrumentationScopeInfo.getVersion() != null) {
-      spanBuilder.putTag(KEY_INSTRUMENTATION_SCOPE_VERSION, instrumentationScopeInfo.getVersion());
-      // Include instrumentation library name for backwards compatibility
-      spanBuilder.putTag(
-          KEY_INSTRUMENTATION_LIBRARY_VERSION, instrumentationScopeInfo.getVersion());
-    }
+  private final Function<SpanData, Span> otelToZipkin;
```

Comment:
#nit: can we be consistent with the language we use to refer to this? In different places its called "otelToZipkin", "otelToZipkinTransformer", and "transformer". 

How about just "transformer", or "mapper"?

Reply:
It's consistent within this class. But I can just name those `Function` vars to `transformer`.